frontend/src/components/common/form/FastCurrencyInput.tsx

```diff
@@ -124,7 +124,16 @@ const CurrencyInput = ({
           name={field}
           onChange={(e: any) => {
             const cleanValue = e.target.value.replace(/[^0-9.]/g, '');
-            setFieldValue(field, cleanValue ? parseFloat(cleanValue) : '');
+            let value: string | number = '';
+            if (cleanValue) {
+              const cleanValueArr = cleanValue.split('.');
+              if (cleanValueArr.length > 1 && cleanValueArr[1].length >= 2) {
+                value = parseFloat(cleanValue).toFixed(2);
+              } else {
+                value = parseFloat(cleanValue);
+              }
+            }
+            setFieldValue(field, value);
```

Comment:
This function will now set the field value to either a string - e.g. '1000.00' or a number - e.g. 1000 depending on what you type (1000.00 will get a string, 1000 will get a number)

I think this should always return a number or a string because consumers of this component would not expect to receive a string in some cases. What are your thoughts @devinleighsmith ?

Reply:
To make it a bit clear. What I'm saying is that branch that does `parseFloat().toFixed(2)` returns a string while the else branch with just `parseFloat()` returns a number. I'm fine if we make both strings but should be consistent.

Reply:
Thanks for your feedback, I also thought we shall format the value to two decimal places always to be more consistent like below, please let me know your thoughts so that I can push it
![image](https://user-images.githubusercontent.com/102620827/166400085-bb691491-c055-4d38-b5da-b0dd96b2512d.png)